lib/charms/opensearch/v0/helper_security.py

```diff
@@ -136,58 +138,124 @@ def split_ca_chain(pem_content: str) -> list[str]:
     return [f"{part}\n{end_cert_marker}" for part in parts]
 
 
-def store_ca(
-    alias: str, store_pwd: str, store_path: str, ca: str, keep_previous: bool = True
+def _store_ca_chain(  # noqa: C901
+    *,
+    alias_base: str,
```

Comment:
For clarity, the client code should only know about an alias not what happens internally:
```suggestion
    alias: str,
```

Reply:
Done.